tour_model_eval/trips_in_bins_and_clusters.ipynb

```diff
@@ -130,138 +140,184 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "latest-cooling",
+   "id": "systematic-huntington",
    "metadata": {
     "scrolled": false
    },
    "outputs": [],
    "source": [
-    "bin_start_lat = []\n",
-    "bin_start_log = []\n",
-    "\n",
-    "# Flip indices because points are in geojson (i.e. lon, lat)\n",
-    "for trip in bin_trips:\n",
-    "    trip_start_lat = trip.data.start_loc[\"coordinates\"][1]\n",
-    "    trip_start_log = trip.data.start_loc[\"coordinates\"][0]\n",
-    "    bin_start_lat.append(trip_start_lat)\n",
-    "    bin_start_log.append(trip_start_log)"
+    "# bin_trips"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "international-unemployment",
+   "id": "public-robin",
    "metadata": {},
    "outputs": [],
    "source": [
-    "bin_end_lat = []\n",
-    "bin_end_log = []\n",
-    "for trip in bin_trips:\n",
-    "    trip_end_lat = trip.data.end_loc[\"coordinates\"][1]\n",
-    "    trip_end_log = trip.data.end_loc[\"coordinates\"][0]\n",
-    "    bin_end_lat.append(trip_end_lat)\n",
-    "    bin_end_log.append(trip_end_log)"
+    "# bins"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "systematic-huntington",
+   "id": "amber-schema",
    "metadata": {
     "scrolled": false
    },
    "outputs": [],
    "source": [
-    "# bin_trips"
+    "# Plot all the bin trips on the map, later we should use different color for different bins, but not sure whether it works.\n",
+    "m = folium.Map(location=[bin_trips[0].data.start_loc[\"coordinates\"][1], bin_trips[0].data.start_loc[\"coordinates\"][0]],zoom_start=12, max_zoom= 30, control_scale=True)\n",
+    "t_index = 0\n",
+    "for bin in bins:    \n",
+    "    for i in range (len(bin)):\n",
+    "        layer = folium.PolyLine([[bin_trips[t_index].data.start_loc[\"coordinates\"][1],bin_trips[t_index].data.start_loc[\"coordinates\"][0]],\n",
```

Comment:
can you add a comment here about why you are using `t_index` instead of `i` for the plotting?
I can figure it out (you want all the trips in the same bin to be the same color eventually) but as you have found from reading Naomi's code, it is very helpful to have comments in the code that explain your thought process to the next reader.

Reply:
Actually, from Naomi's code and the output, the `bin_trips`is filtered data, but they don't follow the index in bins. I think she just appended the filtered data in the new data list, so they don't follow `i`.

Reply:
I am not sure what you mean by that. Can you give an example? And maybe add the example to the comment as well.